addons/binding/org.openhab.binding.zoneminder/ESH-INF/thing/monitor-thing.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="zoneminder" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Supported ZoneMinder devices and features -->
+    <thing-type id="monitor">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="server" />
+        </supported-bridge-type-refs>
+
+        <label>ZoneMinder Monitor</label>
+        <description>Camera in ZoneMinder</description>
+
+        <channels>
+            <channel id="name" typeId="monitor_name" />
+            <channel id="enabled" typeId="monitor_enabled" />
+            <channel id="sourcetype" typeId="monitor_sourcetype" />
+            <channel id="trigger-event" typeId="monitor_trigger_event" />
+            <channel id="function" typeId="monitor_function" />
+            <channel id="is-alive" typeId="monitor_is_alive" />
+            <channel id="daemon-capture-state" typeId="monitor_zmc_daemon_state" />
+            <channel id="daemon-capture-statustext" typeId="monitor_zmc_daemon_statustext" />
+            <channel id="daemon-analysis-state" typeId="monitor_zma_daemon_state" />
+            <channel id="daemon-analysis-statustext" typeId="monitor_zma_daemon_statustext" />
+            <channel id="daemon-frame-state" typeId="monitor_zmf_daemon_state" />
+            <channel id="daemon-frame-statustext" typeId="monitor_zmf_daemon_statustext" />
+            
+        </channels>
+
+        <config-description-ref uri="thing-type:monitor-channels:config" />
+    </thing-type>
+
+
+    <!-- Channel definitions of ZoneMinder Server -->
+    <channel-type id="monitor_id">
```

Comment:
OK. Will it be possible to retrieve this value from eg. a rule? I also plan to have an action where I could query each monitor. There i would need the Monitor Id. But for now It would be fine just to remove the channel.